### PR TITLE
Update some broken links of docs

### PR DESCRIPTION
**Pull request by ntnguyencse**
I'm learning and researching about the Nephio project, but I have some problems with outdated docs.

### CODE REVIEW

**Review by CsatariGergely — APPROVED** [review]

**Comment by ntnguyencse** [comment]
Hi @CsatariGergely, I understand that it is temporary. Please continue to merge the pull request.

File: README.md [hunk 1]
```diff
@@ -36,15 +36,15 @@ includes a [demo video](https://youtu.be/mFl71sy2Pdc) and short articles about
...
 
 ## User Documentation
-* [Release Notes](https://github.com/nephio-project/docs/blob/main/release-notes/README.md)
-* [Demo Sandbox Environment Installation](https://github.com/nephio-project/docs/blob/main/install-guide/README.md)
-* [Quick Start Exercises](https://github.com/nephio-project/docs/blob/main/user-guide/exercises.md)
-* [User Guide](https://github.com/nephio-project/docs/blob/main/user-guide/README.md)
+* [Release Notes](https://github.com/nephio-project/docs/blob/main/content/en/docs/release-notes)
+* [Demo Sandbox Environment Installation](https://github.com/nephio-project/docs/blob/main/content/en/docs/guides/install-guides/_index.md)
+* [Quick Start Exercises](https://github.com/nephio-project/docs/blob/main/content/en/docs/guides/user-guides/exercises.md)
+* [User Guide](https://github.com/nephio-project/docs/blob/main/content/en/docs/guides/user-guides/_index.md)
```
Comment (electrocucaracha) [inline README.md:42]: I think that we can use relative paths here
```suggestion
* [Release Notes](/content/en/docs/release-notes)
* [Demo Sandbox Environment Installation](/content/en/docs/guides/install-guides/_index.md)
* [Quick Start Exercises](/content/en/docs/guides/user-guides/exercises.md)
* [User Guide](/content/en/docs/guides/user-guides/_index.md)
```

File: README.md [hunk 2]
```diff
@@ -36,15 +36,15 @@ includes a [demo video](https://youtu.be/mFl71sy2Pdc) and short articles about
...
 
 ## Other Documentation
 
 * [Developer Documentation](https://github.com/nephio-project/nephio)
-* [Developer Guide](https://github.com/nephio-project/docs/blob/main/developer-guide/README.md)
+* [Developer Guide](https://github.com/nephio-project/docs/blob/main/content/en/docs/guides/contributor-guides)
```
Comment (electrocucaracha) [inline README.md:47]:
```suggestion
* [Developer Guide](/content/en/docs/guides/contributor-guides)
```

**Review by electrocucaracha — COMMENTED** [review]
@CsatariGergely there is a [GitHub Action](https://github.com/marketplace/actions/markdown-link-check) that we can use to validate the links, do you like to include it?

File: README.md (hunk 1 above)
Comment (ntnguyencse) [reply README.md:42]: Thank @electrocucaracha

**Comment by CsatariGergely** [comment]
> @CsatariGergely there is a [GitHub Action](https://github.com/marketplace/actions/markdown-link-check) that we can use to validate the links, do you like to include it?

Sure we can add it, but for most of the links we are using the "Hugo style" links for what we will need just an other GitHub action :) (https://github.com/marketplace/actions/hugo-broken-link-check)

**Comment by liamfallon** [comment]
Can we pull this PR in and maybe add the Github actions later? This PR fixes the developer guide and install guide links.

**Comment by liamfallon** [comment]
thanks @henderiw !

**Comment by ntnguyencse** [comment]
Thank @henderiw

**Pull request merged**